c/datasetService.c

```diff
@@ -188,6 +188,7 @@ void installDatasetContentsService(HttpServer *server) {
 
   HttpService *httpService = makeGeneratedService("datasetContents", "/datasetContents/**");
   httpService->authType = SERVICE_AUTH_NATIVE_WITH_SESSION_TOKEN;
+  httpService->authorizationType = SERVICE_AUTHORIZATION_TYPE_NONE;
```

Comment:
Do 3rd party plugins have to do this as well and then rebuild the binaries, or will everything work without recompilation?